app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  include Visible
+  belongs_to :article
+
+  VALID_STATUSES = ['public', 'private', 'archived']
+
+  validates :status, inclusion: { in: VALID_STATUSES }
```

Comment:
Is it ok to define this validation here?

Reply:
Same answer from [#1 Comment about Valid Status](https://github.com/SanGeeky/rails-training/pull/1#discussion_r610636657), so i removed